### PR TITLE
Generate a dist to be used by Juju core

### PR DESCRIPTION
**Pull request by hatched**
Using the new flask application format, manually generate a dist to be used by Juju core.

Fixes ubuntudesign/juju-squad#546

#### To QA
- run `./run` & play around, everything should work as expected
- bootstrap a new controller `juju bootstrap google google`
- in another tab build the dist `./run exec yarn run build-dist`
- when both are completed `cd dist; juju upgrade-gui jujugui-2.14.1.tar.bz2`
- visit the new gui `juju gui`
- play around, everything should work as expected.

### CODE REVIEW

File: templates/index.html.go [hunk 1]
```diff
@@ -180,8 +180,8 @@ with this program.  If not, see <http://www.gnu.org/licenses/>.
     <script src="{{.comboURL}}?app/assets/javascripts/version.js"></script>
     <script src="{{.comboURL}}?app/assets/javascripts/yui/yui/yui.js"></script>
     {{else}}
-    <script src="{{.comboURL}}?app/assets/javascripts/version.js"></script>
-    <script src="{{.comboURL}}?app/assets/javascripts/yui/yui/yui-min.js"></script>
+    <script src="{{.staticURL}}/static/build/version.json"></script>
```
Comment (hatched) [inline templates/index.html.go]: Both of these html templates need considerable amount of cleanup - this will be done in a follow-up

**Comment by barrymcgee** [comment]
@hatched ☝🏻 Build error

**Comment by hatched** [comment]
Thanks @barrymcgee but this is a known issue that will be resolved once we switch to circle ci and do a bunch of cleanup in the repo. The issue is basically that the existing CI system requires python 2 but we're using 3

**Comment by hatched** [comment]
Great thanks @anthonydillon

**Pull request merged**